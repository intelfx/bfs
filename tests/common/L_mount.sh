test "$UNAME" = "Darwin" && skip

cd "$TEST"
mkdir foo mnt

bfs_sudo mount -t tmpfs tmpfs mnt || skip
defer bfs_sudo umount mnt

ln -s ../mnt foo/bar
"$XTOUCH" mnt/baz
ln -s ../mnt/baz foo/qux

bfs_diff -L . -mount
