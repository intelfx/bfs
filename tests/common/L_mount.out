.
./foo
./foo/bar
./foo/qux
./mnt
